draft-wang-ppm-dap-taskprov.md

```diff
@@ -448,11 +448,10 @@ request with "invalidTask".
 > this to the Author?
 
 Finally, the Helper completes the request as usual, deriving the VDAF
-verification key for the task as described in {{vdaf-verify-key}}.
-
-During Helper aggregate initialization, if any Helper's report share does not
-include the `taskprov` extension with an empty payload, then the Helper MUST
-abort the aggregate request with "invalidMessage".
+verification key for the task as described in {{vdaf-verify-key}}. For any
+report share that does not include the `taskprov` extension with an empty
+payload, then the Helper MUST mark the report as invalid with error
+"invalid_message" and reject it.
```

Comment:
the rest of the document uses `invalidMessage`,  so does core dap draft.

Reply:
"invalid_message" is a variant of PrepareError, not an abort.